src/Input/Input.jsx

```diff
@@ -105,13 +107,26 @@ export const Input = forwardRef(
     return (
       <StyledFlex flexDirection='column' width={1} {...rest}>
         {label ? (
-          <Typography
-            as='label'
-            htmlFor={id}
-            variant='inputLabel'
-            width='fit-content'>
-            {label}
-          </Typography>
+          <Box>
+            <Typography
+              as='label'
+              htmlFor={id}
+              variant='inputLabel'
+              width='fit-content'>
+              {label}
+            </Typography>
+            {!required && notRequiredText && (
+              <Typography
+                ml={2}
+                as='label'
+                htmlFor={id}
+                variant='inputLabel'
```

Comment:
I would suggest avoiding the old typography variants. Try to use the new `typographyVariants`.

Reply:
I wanted to keep the style consistent with the existing label. Changing both will result into a noticeable design change. Do we want that?

Reply:
If the design is using the new variant, then yes

Reply:
Ok, I made the changes